### ISSUE TITLE
Allow empty catch block

### ISSUE THREAD

**Issue by greghuc** [DanielXMoore/Civet#310]
### Coffeescript bug (parse error)

```
"civet coffeeCompat"

output = 'Hello'

try
  output = 'World'
catch
  # do nothing

console.log(output)
```
gives
```
"civet coffeeCompat"

output = 'Hello'

try
  output = 'World'
catch
  # do nothing

console.log(output)
^ ParseError
```

### Playground links

Coffeescript: https://coffeescript.org/#try:%0A%22civet%20coffeeCompat%22%0A%0Aoutput%20%3D%20'Hello'%0A%0Atry%0A%20%20output%20%3D%20'World'%0Acatch%0A%20%20%23%20do%20nothing%0A%0Aconsole.log(output)%0A
Civet: https://civet.dev/playground/?code=ImNpdmV0IGNvZmZlZUNvbXBhdCIKCm91dHB1dCA9ICdIZWxsbycKCnRyeQogIG91dHB1dCA9ICdXb3JsZCcKY2F0Y2gKICAjIGRvIG5vdGhpbmcKCmNvbnNvbGUubG9nKG91dHB1dCkKCgoKCgo%3D

**Comment by greghuc** [comment]
[Making the catch statement non-empty](https://civet.dev/playground/?code=ImNpdmV0IGNvZmZlZUNvbXBhdCIKCm91dHB1dCA9ICdIZWxsbycKCnRyeQogIG91dHB1dCA9ICdXb3JsZCcKY2F0Y2gKICBudWxsCgpjb25zb2xlLmxvZyhvdXRwdXQpCgo%3D) fixes the issue:

```
"civet coffeeCompat"

output = 'Hello'

try
  output = 'World'
catch
  null

console.log(output)
```

Feels pretty yuck to start with.

**Comment by greghuc** [comment]
This bug is more of an FYI. I fixed it in the Coffeescript codebase I'm exploring porting to Civet by switching to the parsable form.

**Comment by edemaine** [comment]
Agreed that we should allow empty `catch` blocks for CoffeeScript-compatibility.

Meanwhile, note that you can simply remove `catch` altogether:
* [CoffeeScript link](https://coffeescript.org/#try:try%0A%20%20output%20%3D%20'World'%0A)
* [Civet link](https://civet.dev/playground?code=dHJ5CiAgb3V0cHV0ID0gJ1dvcmxkJw%3D%3D)

**Comment by greghuc** [comment]
@edemaine thanks didn't know that about no catch. That seems pretty horrendous too!

**Comment by STRd6** [comment]
Fixed in v0.5.61
